### PR TITLE
Release-5.14.0

### PR DESCRIPTION
**Pull request by AdamL-Microsoft**
## Summary of the Pull Request

_What is this about?_
OneFuzz Release 5.14.0

## Validation Steps Performed

* [ ] Passed integration tests

### CODE REVIEW

**Review by stishkin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.14.0
+### Added
+* Service: Added error message to catch VMSS Model `HttpResponseError` [#2384](https://github.com/microsoft/onefuzz/pull/2384)
```
Comment (nharper285) [inline CHANGELOG.md:9]: Don't need to include. Just error reporting.

**Review by nharper285 — COMMENTED** [review]
General comment - do we want to make some kind of note that we are switching predominately over to c#?

File: CHANGELOG.md (hunk 1 above)
Comment (AdamL-Microsoft) [reply CHANGELOG.md]: Removed

**Review by nharper285 — APPROVED** [review]

**Comment by AdamL-Microsoft** [comment]
> General comment - do we want to make some kind of note that we are switching predominately over to c#?

Based on the mixed feedback on this I think we should wait until we're code-complete for C# and/or making changes to deploy C# only by default.

**Pull request merged**